Snakefile

```diff
@@ -108,3 +147,20 @@ rule build_nmd_file:
         """
         python scripts/build_nmd_file.py {input.pdb_file} {input.dcd_file} {output.nmd_file}
         """
+
+rule get_trajectory_correlation:
+    input:
+        pdb="data/pdb/{protein_name}.pdb",
+        traj="data/traj/{protein_name}.xtc"
+    output:
+        correlation_png="output/correlation/{protein_name}.png",
+        correlation_txt="output/correlation/{protein_name}.txt"
+    shell:
+        """
+        python vendor/mdtask/calc_correlation.py \
+            --trajectory {input.traj} \
+            --topology {input.pdb} \
+            --prefix output/correlation/{wildcards.protein_name} \
+            --step 100 \
+            --lazy-load
+        """
```

Comment:
The correlation file is just a txt with a matrix of numbers from 0 to 1. The png shows a heatmap.